### PR TITLE
Implement MDX Remote

### PR DESCRIPTION
**Pull request by jescalan**
Implements [this RFC](https://docs.google.com/document/d/1Fk_TXUZzBiNIo4fGOhlL_v4kXyToLDgockc2fq0THmU/edit)! - Check [the deploy preview](https://deploy-preview-10345--packer-www.netlify.app/)

### CODE REVIEW

**Review by sylviamoss — APPROVED** [review]
LGTM!! 👍

**Review by azr — APPROVED** [review]
LGTM

File: website/components/_temporary-markdown-page/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+}
+
+export function getStaticProps() {
+  return generateStaticProps({
+    pagePath: 'content/test-page.mdx', // resolved from project root
+  })
+}
+```
+
+If the specified page contains front matter, the `page_title` and `description` keys will be added as the title and description the the `<head>` of the page.
```
Comment (nywilken) [inline website/components/_temporary-markdown-page/README.md:22]:
```suggestion
If the specified page contains front matter, the `page_title` and `description` keys will be added as the title and description to the `<head>` of the page.
```

File: website/content/community-plugins.mdx [hunk 2]
```diff
@@ -0,0 +1,41 @@
+---
```
Comment (nywilken) [inline website/content/community-plugins.mdx:1]: Should these pages be part of the site navigation? I don't see them on packer.io at the moment but they are reachable.

**Review by nywilken — COMMENTED** [review]
This looks good, thanks for opening it up.

I left a question regarding a couple of the community pages which don't seem to be in the site navigation. They are also missing from Packer.io 

I tried testing locally but it looks like we need to update the docker image as well. 

```
sh: next-remote-watch: not found
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! packer-docs@1.0.0 start: `next-remote-watch ./content/**/*.mdx`
npm ERR! spawn ENOENT
npm ERR! 
npm ERR! Failed at the packer-docs@1.0.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2020-12-07T16_25_11_850Z-debug.log
make: *** [Makefile:6: website] Error 1```

File: website/content/community-plugins.mdx (hunk 2 above)
Comment (jescalan) [reply website/content/community-plugins.mdx:1]: I'm not sure - I retained the site's existing structure for this PR but the nav can be changed any time via `website/data/subnav.js` 😀

**Comment by jescalan** [comment]
@nywilken - the docker image can be used locally with this set of changes using `make build-image` followed by `make website-local` -- the remote docker image will update via CI when this merges!

**Comment by SwampDragons** [comment]
Can you take a look at the conflict and let us know what we need to do there?

**Comment by jescalan** [comment]
@SwampDragons just resolved that conflict - this should be ready to roll!

**Comment by SwampDragons** [comment]
Thanks, Jeff!

**Pull request merged**